nixnet/_enums.py

```diff
@@ -2029,3 +2029,26 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+class Terminals(enum.Enum):
+    "Terminals to import/export from the XNET device"
+    NX_TERM_FRONTPANEL0 = _cconsts.NX_TERM_FRONT_PANEL0
+    NX_TERM_FRONTPANEL1 = _cconsts.NX_TERM_FRONT_PANEL1
+    NXTERM_PXI_TRIG0 = _cconsts.NX_TERM_PXI_TRIG0        
+    NXTERM_PXI_TRIG1 = _cconsts.NX_TERM_PXI_TRIG1
+    NXTERM_PXI_TRIG2 = _cconsts.NX_TERM_PXI_TRIG2
+    NXTERM_PXI_TRIG3 = _cconsts.NX_TERM_PXI_TRIG3
+    NXTERM_PXI_TRIG4 = _cconsts.NX_TERM_PXI_TRIG4
+    NXTERM_PXI_TRIG5 = _cconsts.NX_TERM_PXI_TRIG5
+    NXTERM_PXI_TRIG6 = _cconsts.NX_TERM_PXI_TRIG6
+    NXTERM_PXI_TRIG7 = _cconsts.NX_TERM_PXI_TRIG7
+    NXTERM_FRONTPANEL0 = _cconsts.NX_TERM_FRONT_PANEL0      
+    NXTERM_FRONTPANEL1 = _cconsts.NX_TERM_FRONT_PANEL1    
+    NXTERM_PXI_STAR = _cconsts.NX_TERM_PXI_STAR
+    NXTERM_PXI_CLK10 = _cconsts.NX_TERM_PXI_CLK10
+    NXTERM_10MHZTIMEBASE = _cconsts.NX_TERM_10_M_HZ_TIMEBASE
+    NXTERM_1MHZTIMEBASE = _cconsts.NX_TERM_1_M_HZ_TIMEBASE
```

Comment:
What should be the word splitting on these?
- `1MHZ_TIMEBASE`
- `1_MHZ_TIMEBASE
- `1_M_HZ_TIMEBASE

I lean towards: `1_MHZ_TIMEBASE

Reply:
Agreed. I second going with  `#_MHZ_TIMEBASE`.